src/nodes/Image.tsx

```diff
@@ -258,7 +259,7 @@ export default class Image extends Node {
     };
 
   component = (props) => {
-    const { theme, isSelected } = props;
+    const { isSelected } = props;
```

Comment:
Quite a bit has been depreciated since V3 . good thing is most of it works as is without extra params . with the exemption to overlay  (V4 overlayBgColorEnd and V5 classDialog )
V3 to V4 https://github.com/rpearce/react-medium-image-zoom/tree/v4.x#migrating-from-v3-to-v4
V4 To V5  https://github.com/rpearce/react-medium-image-zoom/tree/main#migrating-from-v4-to-v5
Suggestions for theme background  
`classDialog={`rme-image rme-image--background-${theme.backgroundType}`}
`
https://github.com/rpearce/react-medium-image-zoom/blob/43cc2b340221236893dca62e44c10bc9f8c1a3ec/source/Controlled.tsx#L310